isoutils/isoutils.go

```diff
@@ -343,20 +343,27 @@ func mkEfiBoot() error {
 		return err
 	}
 
-	/* Copy all required files to efiboot.img and finally unmount efiboot.img */
-	paths := [][]string{
-		{tmpPaths[clrCdroot] + "/images/initrd.gz", tmpPaths[clrEfi] + "/EFI/Boot/initrd.gz"},
-		{tmpPaths[clrImgEfi] + "/EFI/BOOT/BOOTX64.EFI", tmpPaths[clrEfi] + "/EFI/systemd/systemd-bootx64.efi"},
+	/* Copy EFI files to the cdroot for Rufus support */
+	cmds = [][]string{
```

Comment:
This doesn't need to be a slice of slices

Reply:
change to a single array of commands and remove the for loop as there is only one command now.